ClientLib/src/main/java/com/o3dr/services/android/lib/coordinate/LatLongAlt.java

```diff
@@ -53,7 +76,9 @@ public boolean equals(Object o) {
 
         LatLongAlt that = (LatLongAlt) o;
 
-        if (Double.compare(that.mAltitude, mAltitude) != 0) return false;
+        if ((Double.compare(that.mAltitude, mAltitude) != 0)
+            && (that.mFrame.asInt() != mFrame.asInt()) )
```

Comment:
@billbonney Since `mFrame` is an enum, you can just compare it using `that.mFrame != mFrame`.